content/faq/invites.md

```diff
@@ -0,0 +1,54 @@
+---
+title: How does the invite program work?
+category: getstarted
+---
+
+You can earn LBRY Credits (LBC) for inviting others to use LBRY as well as letting us know who invited you. You can invite new users to LBRY via email or a customized URL on the Invites page of LBRY Desktop or lbry.tv. This page will also show the status of existing invites.
+
+*Note: Referral redemptions are typically limited to 20, but increase as you have more verified subscribers on LBRY. See the [limits section below](#limits)*
+
+## Finding your invite link and status
+To get to the Invites area, click the Account menu in the top right and select **Invites**. You can now customize your invite URL by selecting one of your channels from the drop down. Any of them will work, but the email we send out will include your largest subbed channel. You can also invite a user directly via email.
+
+You will also see all of your invites, along with their status(es) - whether the user has accepted it (signed in) and if it is claimable or not (if the user is reward approved).
+
+![invite](https://spee.ch/7/invites-2.jpeg)
+
+## How does the invite link work?
+
+When a user clicks your invite link, they are greeted with a welcome message and option to create an account. They will also automatically subscribe to your channel.
+
+![url](https://spee.ch/6/invites-4.jpeg)
+
+## When do I get my rewards?
+
+If you have claimable invites, you will see a claim button with the total reward amount on the page. You will not receive rewards for each referral until those users join and use LBRY themselves (see below section regarding claimable invites). This is to combat invite fraud and abuse.
+
+## Claiming your bonus for being invited and setting the inviter
+For a limited time, you can claim a reward for being invited and letting us know which channel did so (so they can earn also). You must set the inviter before claiming your reward.
+
+![invitee](https://spee.ch/b/invite-5.jpg)
+
+## How many LBRY Credits do I earn?
+
+New invites are eligible for a 20 LBC reward (subject to change based on our discretion).
+
+## Is there a limit to the credits I can earn? {#limits}
+
+The default limit for claimable invites is 20. This increases as your channel on LBRY gains more verified subscribers. If you want us to review your account and allow for more redemptions, please first make sure you've claimed your maximum and then [email us](mailto:help@lbry.com) with details about yourself and users you are inviting.
```

Comment:
If you want to be more vague, you can say things like "Your limit grows as you use LBRY more and your channel grows in popularity"